zilliqa/src/api/types/eth.rs

```diff
@@ -34,6 +91,9 @@ pub struct Block {
     pub size: u64,
     pub transactions: Vec<HashOrTransaction>,
     pub uncles: Vec<B256>,
+    pub qc: QuorumCertificate,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub agg_qc: Option<AggregateQc>,
```

Comment:
Placeholder for a slack debate on whether we should be adding extra fields to eth_ APIs ..